mozilla_django_oidc_db/setup_configuration/models.py

```diff
@@ -0,0 +1,56 @@
+from typing import Optional
+
+from django_setup_configuration.fields import DjangoModelRef
+from django_setup_configuration.models import ConfigurationModel
+from pydantic import AnyUrl
+
+from mozilla_django_oidc_db.models import OpenIDConnectConfig
+
+
+class AdminOIDCConfigurationModel(ConfigurationModel):
+
+    # claim_mapping: Optional[str] = None  # JSON
+
+    # Arrays are overridden to make the typing simpler (the underlying Django field is an ArrayField, which is non-standard)
+    username_claim: Optional[list[str]] = DjangoModelRef(
+        OpenIDConnectConfig, "username_claim"
+    )
+    groups_claim: Optional[list[str]] = DjangoModelRef(
+        OpenIDConnectConfig, "groups_claim"
+    )
+    superuser_group_names: Optional[list[str]] = DjangoModelRef(
+        OpenIDConnectConfig, "superuser_group_names"
+    )
+
+    # Endpoints
+    oidc_op_authorization_endpoint: Optional[AnyUrl] = DjangoModelRef(
+        OpenIDConnectConfig, "oidc_op_authorization_endpoint", required=False
+    )
+    oidc_op_token_endpoint: Optional[AnyUrl] = DjangoModelRef(
+        OpenIDConnectConfig, "oidc_op_token_endpoint", required=False
+    )
+    oidc_op_user_endpoint: Optional[AnyUrl] = DjangoModelRef(
+        OpenIDConnectConfig, "oidc_op_user_endpoint", required=False
+    )
```

Comment:
Theses should be (optionally) optional and I do not know how to make them that way.

Either provide these or the discovery end point